src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -8,7 +8,31 @@ public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws CustomException {
+        if (user.getAge() == null || user.getPassword() == null || user.getLogin() == null) {
+            throw new CustomException("User login,password or age is null!"
+                    + user.getAge() + user.getPassword() + user.getLogin());
+        }
+        if (user.getLogin().isEmpty() || (user.getPassword().isEmpty())) {
+            throw new CustomException("User login or password is empty!"
+                    + user.getLogin() + user.getPassword());
+        }
+        if ((user.getLogin().length()) < 6) {
+            throw new CustomException("User login must be at least 6 characters long! - "
```

Comment:
```suggestion
            throw new CustomException("User login must be at least" + CONSTANT_NAME + " characters long! - "
```
Better use variable. If you change the value, the message will be updated automatically (check it everywhere)